### ISSUE TITLE
Should a `.nojekyll` file be generated by default?

### ISSUE THREAD

**Issue by Marv51** [angular-schule/angular-cli-ghpages#160]
I ran into some issues with GitHub Pages a while ago where my already finished project was changed again by GitHub Pages, because it was compiled again with Jekyll (result: .txt assets are just missing)

The solution was simple enough: Just add a `.nojekyll` file to the root of the output.

I am only starting out with angular-cli-ghpages, but I do believe putting that file would be a good idea. I put it always when the uploaded content is not meant to be compiled (again) with Jekyll, which should be every angular project.

---

From the [GitHub Pages documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages): 
> If you publish your site from a source branch, GitHub Pages will use Jekyll to build your site by default. If you want to use a static site generator other than Jekyll, we recommend that you write a GitHub Actions to build and publish your site instead. Otherwise, disable the Jekyll build process by **creating an empty file called .nojekyll in the root of your publishing source**, then follow your static site generator's instructions to build your site locally.

**Comment by JohannesHoppe** [comment]
I agree, this could be a useful feature. We have mentioned the `nojekyll` file once in the readme. But it could be useful to include it by default.
